modules/Stories/Stories.tsx

```diff
@@ -1,22 +1,58 @@
 import type { FunctionComponent } from 'react';
 import Link from 'next/link';
 import type { Story } from '@prezly/sdk';
+import { Category } from '@prezly/sdk/dist/types';
+import { PaginationProps } from 'types';
+import { useInfiniteStoriesLoading } from '@/hooks/useInfiniteStoriesLoading';
+import Pagination from './Pagination';
 
 type Props = {
     stories: Story[];
+    pagination: PaginationProps;
+    category?: Category;
 };
 
-const Stories: FunctionComponent<Props> = ({ stories }) => (
-    <div>
-        {stories.map((story) => (
-            <Link key={story.id} href={`/${story.slug}`} passHref>
-                <a style={{ display: 'block', marginBottom: 20 }}>
-                    <img src={story.thumbnail_url} alt="" />
-                    <span>{story.title}</span>
-                </a>
-            </Link>
-        ))}
-    </div>
-);
+/**
+ * This component contains code for both Infinite Loading and classic Pagination.
+ * You should choose one or another in your theme.
+ */
```

Comment:
:x: 

I'd implement 2 `Stories` components in this file (or better: in separate files, but in this directory) , and allow user to choose which one to export from here:
- `PaginatedStories`
- `InfiniteStories`

And then place a proper, equivalent comment at the bottom of the file, near the `export default` statement. 

Then it would be clear what needs to be done, and easy to choose from both pagination variants.

Reply:
@kamilmielnik sounds good! But wouldn't it complicate the maintenance? E.g. needing to update other stories display logic in two files instead of one.

Reply:
> E.g. needing to update other stories display logic in two files instead of one.

You mean [this](https://github.com/prezly/theme-nextjs-starter/pull/14/files#diff-c1b731297a897b3b464a8b83a96167bcfa43e5ab379f057348fca49a98f12ea3R30-R37)?
How about extracting a component for it (`StoriesList`?), and using it in `PaginatedStories` & `InfiniteStories`? :)

Reply:
Yeah, that sound good!

Reply:
@kamilmielnik I created the two separate components and for now used `PaginatedStories` as default.
Should I maybe add commented-out `InfiniteStories` code on these pages to indicate that's another option?

Reply:
> Should I maybe add commented-out `InfiniteStories` code on these pages to indicate that's another option?
> Or just mention that in README?

Trying to put myself in the position of someone who will clone this repository and go through all files anyway, I think putting this comment in code would be more appropriate. But why not have both (comment in code + README)? :)

Reply:
Done!